src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -26,6 +26,7 @@ const PaymentAgentWithdraw = (() => {
         ddl_agents: '#ddlAgents',
         txt_agents: '#txtAgents',
         txt_amount: '#txtAmount',
+        frm_msg   : '#form-error',
```

Comment:
there's a `txt_desc` also needing to use `frm_msg` but `txt_desc` also doesn't exist. should we remove that block?

Reply:
By removing `txt_desc` block, do you mean this block? 
```js
$(field_ids.txt_desc).off().on('keyup', () => {
                if (isVisible(getElementById('withdrawFormMessage'))) {
                    $(field_ids.frm_msg).setVisibility(0);
                }
            });
```
If so, I'm not sure whether or not we want to remove this. from the commit message, I only can know this is necessary. What do you suggest?

Reply:
Ok, I've checked it. looks like it was a duplicate of what I've done here. I removed it.